terraform/dns/README.md

```diff
@@ -0,0 +1,77 @@
+# DNSimple Terraform DNS Demo
+
+With the DNSimple Terraform Provider, you can manage your DNS infrastructure, including zones and records, directly within Terraform. This lets you leverage the simplicity and power of Terraform's Infrastructure-as-Code design for your DNSimple resources, and coordinate them alongside the other Terraform resources that form your infrastructure.
+
+As an example of the power, simplicity, and functionality of managing DNS using the DNSimple Terraform Provider, we've created this demo that you can review, run, and play around with. We'll implement a basic email inbox service with a simple but relatable design and structure of a full stack application with all the typical infrastructure:
+
+- An S3 bucket, acting as the data store, containing incoming emails collected via SES.
+- An [API service](./api.py), running on EC2 instances, distributed across two regions, that will serve the emails.
+- A simple [static web application](./app.html) that will be stored and served from an S3 bucket, which will present a UI for the data returned by the API.
+
+We'll showcase how, by using the DNSimple Terraform Provider, setting up the required DNS plumbing for this, and almost all other services like it, can enable smoother and more powerful planning, design, and operations of infrastructure:
+
+- DNS records are declared alongside the components and resources they serve, so there's no loss of context or need for disjointed external tools or manual processes.
+- The Terraform resources can be committed to your VCS for centralizing the source of truth, keeping declarations and state in sync, automated deployments, regression testing, policy enforcement, and inspecting the history of changes.
+- Leverage the full power of DNSimple's functionality and tools, like ALIAS and regional records, to empower your architecture.
+- Take advantage of all of Terraform's benefits with DNS records that are just like any other resource: drift detection, one-command deployment and synchronization, clean and structured deletion, automatic dependency resolution, track and centralize state, and more.
+
+Feel free to use this proof-of-concept as the starting point for your next project, or as inspiration to swap parts out to build something completely different.
+
+## Prerequisites
+
+To run this demo, you will need:
+
+- An authoritative zone on DNSimple, where the DNS resolution is not delegated or disabled. If you purchased or transferred your domain to DNSimple, or you have pointed your domain's nameservers to DNSimple at your registrar, you should be good to go.
+  - If you'd like to registrar and set up a domain also entirely through Terraform, check out our [domains demo](../domains).
+- An API token for your DNSimple account must be generated and stored in your environment variables. See [here](https://support.dnsimple.com/articles/api-access-token/) for more details.
```

Comment:
This mentions the need to store the token in the variables but doesn't say what variables to store the token. Also, the account needs to be added as an env variable for the provider to be authenticated.

I think having these instructions of what exactly to setup in the terminal as part of the **Running** section would be helpful.